packages/engine/src/avatar/systems/AvatarTransparencySystem.tsx

```diff
@@ -100,6 +101,8 @@ export const AvatarTransparencySystem = defineSystem({
 
 const AvatarReactor = (props: { entity: Entity }) => {
   const entity = props.entity
+  if (!entity || !entityExists(entity)) return null
```

Comment:
this should be impossible, subreactors to a query will always have non-zero entity passed in

Reply:
The error was actually coming from the line just below this:
`const sourceID = UUIDComponent.getAsSourceID(entity)`

So instead I now check if the avatar entity has a UUIDComponent and return null if it does not:
`if (!hasComponent(entity, UUIDComponent)) return null`